documentation.md

```diff
@@ -0,0 +1,14 @@
+# Documentation
+This guide aims to make it clear what information can and cannot be included in documents and designs that are made public.
+
+## Restricted Items
+1. Network ranges and IP addresses <br>
```

Comment:
I prefer using the `addresses\` style instead of embedding HTML in MarkDown.  I've not tested what happens with a .md to .pdf converter, but it might not do the right thing with the `<br>` tag.  It's also possible to use two spaces at the end of a line but our a) that's not visible to a reviewer's eye and b) most of our editor's will end up stripping trailing spaces off the end of lines on save.